packages/elements/src/list/__test__/list.test.js

```diff
@@ -279,5 +279,21 @@ describe('list/List', () => {
     el.data.setItemPropertyValue(data[1], 'hidden', false);
     await elementUpdated(el);
   });
+
+  it('Should have focus state remain at host when tapping in an item', async () => {
+    const el = await fixture('<ef-list></ef-list>');
+    el.data = data;
+    await elementUpdated(el);
+
+    const firstElement = el.firstElementChild;
+
+    await triggerFocusFor(el);
+
+    setTimeout(() => firstElement.dispatchEvent(new Event('tap')));
```

Comment:
You should be able to use `el.click()` - this should fire tap event.

Reply:
Updated.